.github/README.md

```diff
@@ -7,35 +7,23 @@
 **Lawnicons** is an icon pack developed by the Lawnchair team.
 Originally an addon for Lawnchair 12 Alpha 5 and above to implement themed icons, it can now be used on many launchers. Some launchers may also support Lawnicons for themed icons.
 
-Lawnicons is best used on Lawnchair 12.1. You can enable themed icons on Lawnchair by going to `Home Settings > General > Icon Style > Themed Icons`, choosing "Home Screen" or "Home Screen and App Drawer".
+Lawnicons is best used on [Lawnchair 12.1](https://github.com/LawnchairLauncher/lawnchair). You can enable themed icons on Lawnchair by going to `Home Settings > General > Icon Style > Themed Icons`, choosing "Home Screen" or "Home Screen and App Drawer".
 
-This project is mostly supported by our community. If you wish to contribute, see the [Contributing section](README.md#Contributing) below.
+This project is mostly supported by our community. If you wish to contribute, check out the Contributing section below.
 
 ## Download
 See the [Releases section](https://github.com/LawnchairLauncher/lawnicons/releases) to download the latest stable build of Lawnicons.
 
-For the development versions with new icons, go to [nightly.link](https://nightly.link/LawnchairLauncher/lawnicons/workflows/build_debug_apk/develop/Debug%20APK).
+For the development versions with new icons, go to [the nightly.link](https://nightly.link/LawnchairLauncher/lawnicons/workflows/build_debug_apk/develop/Debug%20APK).
 
 ## Contributing
 
 Please see the [contributing guide](CONTRIBUTING.md) for information on adding icons or contributing code.
 
-For users familiar with the guidelines, check out [how to use icontool.py](/.github/icontool_guide.md) to speedup the process of adding icons.
+If you know how to create proper icons already, you can use [icontool.py](/.github/icontool_guide.md) to speed up the process of adding icons. Also see the [Figma contributing guide](https://www.figma.com/community/file/1227718471680779613).
 
 ## Requesting icons
 
 To request new icons, please use the [icon request form](https://forms.gle/Fx8vZAiWdW1Tyjo57).
 
-If you want to see the requested icons (for contributing purposes), you can view the form responses in our 
-[publicly available Google Sheets](https://docs.google.com/spreadsheets/d/1h3eiJnG2nEdR1DbvemaF1lYthHkzYbXvVFPP0TEEt5k/edit?usp=sharing) file.
-
-**Note that we currently don't show whether an icon request has been completed or not.**
-
-## License
-
-Lawnicons is licensed under the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0). See [LICENSE](LICENSE) for additional information.
```

Comment:
This shouldn't be removed. While there is a License File, it is still nice to see the license at a glance.